### DISCUSSION TITLE
Thanks for all the time

### DISCUSSION THREAD

**Discussion by scorpion421** [d4rken-org/sdmaid-se#58] [General]
I purchased SD Maid on the day it got released. 2011... a long time ago. You kept/keep the app alive. 12 Years now. SD Maid is the dinosaur of cleaning apps and still the best one, because trustworthy and fulfills what it's about to do. As soon as I saw you put together a new, revived (can I say that? :D) app, I purchased it straight away. It was long overdue to say thanks.
I don't use it that often anymore, to be honest, but it's on every phone I had since then. Again: Thanks for all and for all the time you invest to make SD Maid the queen of all cleaners. 

Warum sagt mir keiner, dass ich auf auf Deutsch schreiben kann 😂. Egal. Passt so. Dann auf Deutsch nochmals vielen lieben Dank für all die Jahre, in welchen du deine Zeit und Mühen in diese App gesteckt hast. Ich denke, SD Maid is eine der letzten Apps aus dieser Zeit, die noch aktiv weiterentwickelt werden. Nova Launcher wurde ja leider auch letztes Jahr im Sommer/Herbst verkauft. Irgendwie kommen die seit knapp einem halben Jahr... Offtopic. Auf hoffentlich viele weitere Jahre 😊.

P.S.: What does the SE in SD Maid SE stand for?
What does the SD in SD Maid stand for? 🤷

Ich sehe, was du da gemacht hast 😂 Ja, SD-Karten sind leider aussterbend bei Smartphones. Zumindest denke ich, dass der Name damals davon herrührte. Heute jedoch aufgrund der Beschränkungen und schwindenden Speicherkarten... tja. Schade. SE steht 100% für "Second Edition", oder liege ich falsch 😊?

**Comment by M-u-m-p-i-t-z** [comment]
Where can you get the SE version as Pro?
Or is that already included with version one as Pro?
